src/app/core/components/left-nav/left-nav.component.ts

```diff
@@ -47,6 +48,10 @@ export class LeftNavComponent implements OnInit, OnDestroy {
       map(content => (content !== null && (isItemInfo(content) || isGroupInfo(content)) ? content : undefined)),
       pairwise(),
     ).subscribe(([ prevContent, content ]) => {
+      if (content?.route.id) {
+        this.selectId.emit(content.route.id);
+      }
```

Comment:
If can/should probably be move in the next block (L57) as otherwise the scroll will be called several time, including when some metadata about the current selection (e.g. its score) changes without sourced by a user click in the left menu.

Reply:
Fixed